project_resource_calendar/tests/test_calendar_event.py

```diff
@@ -42,3 +42,36 @@ def test_010_onchange_room_id(self):
         self.calendar_event._onchange_room_id()
         self.assertEqual(self.calendar_event.room_id.name,
                          'Test Room id after onchange method execution')
+
+    def test_020_calculate_recurrent(self):
+        self.recurrency = True
```

Comment:
the recurrent_state field should be calculated automatically when recurrency field is updated

Reply:
@rimbendhaou: But for testing, I purposely set the value for testing both the scenarios otherwise what is your comment to change in the existing code.